src/app/store/auth/actions/auth.actions.ts

```diff
@@ -0,0 +1,9 @@
+import { createAction, props } from '@ngrx/store';
+
+import { LoginData } from '@app/interfaces/parameters';
+import { AuthState } from '@app/store/auth';
+
+export const retrieveInitialAuthData = createAction('[Auth] Retrieve initial data');
+export const setAuthData = createAction('[Auth] Set data', props<AuthState>());
+export const loginStart = createAction('[Auth] Login start', props<LoginData>());
+export const logoutStart = createAction('[Auth] Logout start');
```

Comment:
as you don't have a logout end action I'd suggest keeping it `logout`. Same for login.
Perhaps, you can specify `userLogin` and `userLogout`, but won't change its purpose..

Reply:
Agree